### PR TITLE
Add Wacom HID 5115 support

### PR DESCRIPTION
**Pull request by jschmidt-1**
Added descriptions for the tablet and the common Thinkpad Pro Pen

### CODE REVIEW

File: data/isdv4-5115.tablet [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# this is for the Wacom pen + touchscreen as found in Lenovo Thinkpad X1 Tablet Gen 2
+
+[Device]
+Name=Wacom HID 5115
+DeviceMatch=i2c:056a:5115
+Width=10
+Height=7
+IntegratedIn=Display;System
+styli=0x21;
```
[the remark this answers is not in the document]
Comment (jschmidt-1) [reply data/isdv4-5115.tablet]: This has been updated in commit https://github.com/linuxwacom/libwacom/pull/154/commits/6bbe63240a886cdcedbf15686e9ea3a758dcf070
Thanks for letting me know.

**Comment by jschmidt-1** [comment]
I updated the definitions and a test case to pass all checks. Please review the changes.
Thank you!

**Comment by jschmidt-1** [comment]
It is the least I can do. Squashed as requested. Thank you guys! :-)

File: data/libwacom.stylus [hunk 2]
```diff
@@ -548,3 +548,10 @@ Type=Puck
 HasLens=false
 HasWheel=true
 Buttons=5
+
+[0x21]
```
[the remark this answers is not in the document]
Comment (jigpu) [reply data/libwacom.stylus]: That's the basic idea/goal. I'm trying to gather some more information first though. I've received a list of pens and their IDs/features, but there seem to be some conflicts that I need to understand. We might need to handle pen IDs differently in libwacom or the kernel...

[the remark this answers is not in the document]
Comment (whot) [reply data/libwacom.stylus]: fair enough. Let's merge this one for now then until we have a solution for the grouping, I don't see a good reason to delay this at this point.

**Pull request merged**